tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature

```diff
@@ -102,3 +102,27 @@ Feature: spaces management
       | username    | anna             |
       | displayname | Anna Murphy      |
       | email       | anna@example.org |
+
+
+  Scenario: assigned groups
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    And "Admin" creates following groups using API
+      | id       |
+      | sales    |
+      | finance  |
+      | security |
+    And "Admin" adds user to the group using API
+      | user  | group |
+      | Alice | sales |
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    When "Admin" adds the user "Alice" to the groups "finance,security"
```

Comment:
Rather than a group provided in string i would prefer to use table. Its your call though. :+1:

Reply:
I left it as it is in the example of step with batch action